app/models/user.rb

```diff
@@ -1,2 +1,9 @@
 class User < ApplicationRecord
+  has_many :posts, foreign_key: :author_id
+  has_many :comments
+  has_many :likes
```

Comment:
- [ ] Please add the foreign key `author_id` to both the comments and likes. It should look something like this
```
has_many :comments, foreign_key: 'author_id'
has_many :likes, foreign_key: 'author_id'

```

Reply:
Thank you, that solution is better, I am applying